tools/functional-tester/etcd-runner/command/election_command.go

```diff
@@ -65,8 +68,8 @@ func runElectionFunc(cmd *cobra.Command, args []string) {
 				break
 			}
 		}
-		e := concurrency.NewElection(s, "electors")
 
+		e := concurrency.NewElection(s, elector)
```

Comment:
Passing in an election name to etcd-runner would be good. It can be through args[0] in the command. The functional tester can name each instance with the current nano time.